alvr/server/gen_loader.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
You should accept POSIX compliant shells:`#!/bin/sh`

Reply:
done